src/components/UiKit/Icons/HeaderLogo/index.jsx

```diff
@@ -0,0 +1,20 @@
+export default function HeaderLogoIcon() {
```

Comment:
Consider combining HeaderLogo, HeaderSubText and Language icon into a single Header folder for better organization as well.